File: concurrency_paralellism/concurrency/hello_word.go
package main

import (
	"fmt"
	"time"
)

func hello() {
	for i := 0; i < 5; i++ {
		fmt.Println("Hello from goroutine!", i)
		time.Sleep(500 * time.Millisecond)
	}
}

func main() {
	go hello()

	for i := 0; i < 3; i++ {
		fmt.Println("Hello from main!", i)
		time.Sleep(1000 * time.Millisecond)
	}
}

File: concurrency_paralellism/concurrency/hello_word.rs
use std::thread;
use std::time::Duration;

fn main(){
	let handle=thread::spawn(|| {
		for i in 1..=5000000{
			println!("Hello from thread! {}", i);
			thread::sleep(Duration::from_millis(1));
		}
	});
		for i in 1..=3{
			println!("Hello from main! {}", i);
			thread::sleep(Duration::from_millis(1000));
		}
		handle.join().unwrap();
}